### PR TITLE
correct errors in inp files

### PR DESCRIPTION
**Pull request by mickaelaccensi**
remove -N for and ww3_ta1 ans edit ww3_tp2.14

# Pull Request Summary
correct errors in inp files in regtests and remove unecessary -N in matrix.base

## Description
correct errors in inp files in regtests and remove unecessary -N in matrix.base

### Issue(s) addressed
no issues adressed

### Commit Message
correct errors in inp files in regtests and remove unnecessary -N in matrix.base

### Check list  

- [x] Branch is up to date with the authoritative repository (NOAA-EMC) develop branch. 
- [x] Checked the [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop). 
- [ ] If a version number update is required, checked the [updating version number](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number) checklist. 
- [ ] If a new feature was added, a regression test for testing the new feature is added. 
 
### Testing

* How were these changes tested?
* Are the changes covered by regression tests? (If not, why? Do new tests need to be added?)
* Have the matrix regression tests been run (if yes, please note HPC and compiler)?
* Please provide the summary output of matrix.comp (_matrix.Diff.txt_, _matrixCompFull.txt_ and _matrixCompSummary.txt_):    
* Please indicate the expected changes in the regression test output ([Note the known list of non-identical tests](https://github.com/NOAA-EMC/WW3/wiki/How-to-use-matrix.comp-to-compare-regtests-with-master#4-look-at-results)).
* Please provide the summary output of matrix.comp (_matrix.Diff.txt_, _matrixCompFull.txt_ and _matrixCompSummary.txt_):

### CODE REVIEW

File: regtests/bin/matrix.base [hunk 1]
```diff
@@ -2055,18 +2055,18 @@
  if [ "$assim" = 'y' ]
  then
    echo ' ' >> matrix.body
-   echo "$rtst -s ST4 -N -w work_UPD0F_O    -i input_UPD0F_O    $ww3 ww3_ta1" >> matrix.body
```
Comment (JessicaMeixner-NOAA) [inline regtests/bin/matrix.base:2058]: I understand that we needed to update the .inp files, but isn't it okay to leave the -N files for the few cases where it's explicitly requested?  I know that there is at least one feature that was only implemented in .nml files for example and so for that test I'd think we'd want to keep the -N option.

File: regtests/bin/matrix.base [hunk 2]
```diff
@@ -2207,7 +2207,7 @@
   if [ "$confignc" = 'y' ] && [ "$shrd" = 'y' ]
   then
     echo ' ' >> matrix.body
-    echo "$rtst -N $ww3 ww3_tnc1" >> matrix.body
```
Comment (JessicaMeixner-NOAA) [inline regtests/bin/matrix.base]: @ukmo-ccbunney can you confirm that this test needs the -N option or not?

File: regtests/bin/matrix.base (hunk 1 above)
Comment (aliabdolali) [reply regtests/bin/matrix.base:2058]: Even if we keep -N option, I still would like the inp files consistent with nml. So, this PR should be committed, preferably with all without -N for the sake of consistency.

**Comment by aliabdolali** [comment]
@mickaelaccensi I made a PR to your branch with a fix for tnc1 test. Could you review and merge?

**Comment by mickaelaccensi** [comment]
> @mickaelaccensi I made a PR to your branch with a fix for tnc1 test. Could you review and merge?

done

**Comment by mickaelaccensi** [comment]
@aliabdolali can you merge it or do I need to change something else ?

**Comment by aliabdolali** [comment]
@mickaelaccensi could you revert 50 to 10 in ww3_grid.inp for all ww3_ta1 cases? I am not sure how ww3_grid.nml works with 50, but with inp, the ww3_grid execution fails.

**Comment by mickaelaccensi** [comment]
> @mickaelaccensi could you revert 50 to 10 in ww3_grid.inp for all ww3_ta1 cases? I am not sure how ww3_grid.nml works with 50, but with inp, the ww3_grid execution fails.

done

**Comment by aliabdolali** [comment]
@mickaelaccensi thanks, i am going to run the tests.

File: regtests/bin/matrix.base (hunk 2 above)
Comment (ukmo-ccbunney) [reply regtests/bin/matrix.base]: Yes - in this case the regtest is testing features that are only available via the namelist version of the ww3_ounf input file.

**Comment by mickaelaccensi** [comment]
I've reverted matrix.base for ww3_tnc1 to use namelist by default. 
@aliabdolali , I think you can now merge this branch

**Comment by aliabdolali** [comment]
The test ran successfully on NOAA rdhpc with the intel compiler.
The preknown non-identical cases:


```
**********************************************************************
********************* non-identical cases ****************************
**********************************************************************
mww3_test_03/./work_PR2_UQ_MPI_d2                     (15 files differ)
mww3_test_03/./work_PR2_UNO_MPI_d2                     (12 files differ)
mww3_test_03/./work_PR3_UQ_MPI_d2_c                     (15 files differ)
mww3_test_03/./work_PR1_MPI_d2                     (18 files differ)
mww3_test_03/./work_PR3_UNO_MPI_d2_c                     (12 files differ)
mww3_test_03/./work_PR3_UNO_MPI_d2                     (12 files differ)
mww3_test_03/./work_PR3_UQ_MPI_d2                     (15 files differ)
ww3_tp2.10/./work_MPI_OMPH                     (7 files differ)
ww3_tp2.16/./work_MPI_OMPH                     (4 files differ)
ww3_ufs1.3/./work_a                     (1 files differ)
```

and changes only in stdouts for the following cases (expected).
```
ww3_tp2.14/./work_OASACM6                     (8 files differ)
ww3_ta1/./work_UPD2_U                     (4 files differ)
ww3_ta1/./work_UPD5_U                     (4 files differ)
ww3_ta1/./work_UPD0F_O                     (4 files differ)
ww3_ta1/./work_UPD3_U_cap                     (4 files differ)
ww3_ta1/./work_UPD6_O                     (4 files differ)
```

[matrixCompFull.txt](https://github.com/NOAA-EMC/WW3/files/8548956/matrixCompFull.txt)
[matrixCompSummary.txt](https://github.com/NOAA-EMC/WW3/files/8548957/matrixCompSummary.txt)
[matrixDiff.txt](https://github.com/NOAA-EMC/WW3/files/8548958/matrixDiff.txt)

**Review by aliabdolali — APPROVED** [review]

**Pull request merged**